phi/tools/giphy.py

```diff
@@ -0,0 +1,70 @@
+import uuid
```

Comment:
Lets please use PEP 8 guidelines for import order:
```
import os
import uuid
from typing import Optional

import httpx
```

Reply:
I'll add a script to automatically fix import order?

Reply:
It will change the import order for ALL files most likely. Should do it as part of cleanup for v3

Reply:
I fixed this one manually as per your request.